### ISSUE TITLE
FR: Control through USB-C

### ISSUE THREAD

**Issue by Smulee** [bigtreetech/PandaPWR#18]
I prefer to use wired connections instead of wireless.

I do believe that control through the USB-C port is already possible with serial commands, if not, it should be developed.

**Comment by zubir2k** [comment]
I second this.. either usb direct to pc or using usb-c to Ethernet

**Comment by avnc** [comment]
Yes, third vote. Making this triggerable from wired connection would be great.

**Comment by bigtreetech** [comment]
This is a good idea guys. We like it!

**Comment by Smulee** [comment]
> This is a good idea guys. We like it!

Glad to hear it.

Are there any updates on this matter?

By the way, I was quite disappointed when my unit did not have a debug duck as stated in the manual. You let me down

I have since gotten back on my feet and love the unit :)
